sdk/storage/generate.ps1

```diff
@@ -1,16 +1,17 @@
 pushd $PSScriptRoot/Azure.Storage.Common/swagger/Generator/
 npm install
+npm install -g autorest@beta
 
 cd $PSScriptRoot/Azure.Storage.Blobs/swagger/
-npx autorest --use=$PSScriptRoot/Azure.Storage.Common/swagger/Generator/ --verbose
+autorest --use=$PSScriptRoot/Azure.Storage.Common/swagger/Generator/ --verbose
```

Comment:
Shouldn't this be `autorest-beta`? @MiYanni

Reply:
If I'm understanding which version they are using, they are using the one we made for PowerShell, which wasn't `autorest-beta` yet.

Reply:
Using `autorest-beta` doesn't work.

Reply:
OK I get confused by "autorest@beta" vs "autorest-beta" :) I keep thinking they should be the same but they aren't.

Reply:
Yeah. `autorest@beta` uses the `autorest` command while `@autorest/autorest` uses `autorest-beta`. I know... but this stuff is temporary.